source/_integrations/ecovacs.markdown

```diff
@@ -55,6 +55,7 @@ Additionally, **depending on your model**, the integration provides the followin
   - `Map`: The floorplan/map as an image in SVG format.
 - **Number**:
   - `Clean count`: Set the number of times to clean the area.
+  - `Cut direction`: Set the mower cutting direction.
```

Comment:
could you give an example? I don't understand how a number can tell me the direction.

Reply:
Sorry, didn't see the notification requesting a change here.  I've added `(from 0 to 180 degrees)` as that hopefully makes it clearer that this is the the range in direction of travel for the mower.

The terminology I think is hopefully apparent for an owner of the mower where the term "direction" is already used in the Ecovacs app and is accompanied by a little drag arrow across the 180 degree arc for orientation.